local-volume/utils/update-pv-to-beta/README.md

```diff
@@ -0,0 +1,140 @@
+Annotations:     volume.alpha.kubernetes.io/node-affinity={ "requiredDuringSchedulingIgnoredDuringExecution": { "nodeSelectorTerms": [ { "matchExpressions": [ { "key": "kubernetes.io/hostname", "operator": "In", "valu...
+Finalizers:      [kubernetes.io/pv-protection]
+StorageClass:    local-storage
+Status:          Available
+Claim:
+Reclaim Policy:  Retain
+Access Modes:    RWO
+Capacity:        200Mi
+Node Affinity:   <none>
+Message:
+Source:
+    Type:  LocalVolume (a persistent volume backed by local storage on a node)
+    Path:  /mnt/disks/vol/vol1
+Events:    <none>
+```
+
+### Create ServiceAccount and kubernetes job to update local PV alpha node affinity to beta
+``` console
+kubectl create -f deployment/kubernetes/admin-account.yaml
+kubectl create -f deployment/kubernetes/update-pv-to-beta.yaml
+```
+
+### Describe the kubernetes job to see if it succeeds
+``` kubectl get job
+kubectl get job
+NAME                   DESIRED   SUCCESSFUL   AGE
+local-volume-updater   1         1            10s
+```
+``` kubectl describe job local-volume-updater
+kubectl describe job local-volume-updater
+Name:           local-volume-updater
+Namespace:      default
+Selector:       controller-uid=c2a02fe4-3641-11e8-afd6-080027765304
+Labels:         app=local-volume-updater
+Annotations:    <none>
+Parallelism:    1
+Completions:    1
+Start Time:     Mon, 02 Apr 2018 14:47:50 +0800
+Pods Statuses:  0 Running / 1 Succeeded / 0 Failed
+Pod Template:
+  Labels:           controller-uid=c2a02fe4-3641-11e8-afd6-080027765304
+                    job-name=local-volume-updater
+  Service Account:  local-storage-update
+  Containers:
+   updater:
+    Image:        quay.io/external_storage/local-volume-update-pv-to-beta:latest
+    Port:         <none>
+    Host Port:    <none>
+    Environment:  <none>
+    Mounts:       <none>
+  Volumes:        <none>
+Events:
+  Type    Reason            Age   From            Message
+  ----    ------            ----  ----            -------
+  Normal  SuccessfulCreate  20s   job-controller  Created pod: local-volume-updater-lb5zs
+```
+if error occurs, we can use `kubectl get pods` and `kubectl logs $podID` to see the error log and debug
+
+### Describe the local PV again to check if the alpha node affinity is updated
+``` kubectl describe pv example-local-pv-1
+Name:              example-local-pv-1
+Labels:            <none>
+Annotations:       <none>
+Finalizers:        [kubernetes.io/pv-protection]
+StorageClass:      local-storage
+Status:            Available
+Claim:
+Reclaim Policy:    Retain
+Access Modes:      RWO
+Capacity:          200Mi
+Node Affinity:
+  Required Terms:
+    Term 0:        kubernetes.io/hostname in [nickren-14]
+Message:
+Source:
+    Type:  LocalVolume (a persistent volume backed by local storage on a node)
+    Path:  /mnt/disks/vol/vol1
+Events:    <none>
+```
+
+### Delete ServiceAccount and kubernetes job
+``` console
+kubectl delete -f deployment/kubernetes/admin-account.yaml
```

Comment:
I believe deletion does not take yaml files as input.  It takes the object type and object name.